chromeos.kernelci.org

```diff
@@ -6,6 +6,18 @@
 
 set -e
 
+# Trap function for errors
+crash() {
+    local exit_code="$?"
+    # If exit code is 0, then it's not an error
+    [ "$exit_code" -eq 0 ] && exit 0
+    # Generate syslog message
+    logger -t kernelci-deploy "Error in script $0"
+    exit $exit_code
```

Comment:
Some shells propagate the exit code even if you have a trap, but I'm not sure if it's a standard feature like POSIX.  That's why I didn't mention it in the previous review, I thought maybe you knew about this?

Reply:
Better safe than sorry :)
As i know it is recommended to put exit, for example:
```
Command exit should be explicitly invoked at the end of the command list, otherwise the commands that follow trap in the script could execute in an undefined state.
```